specification/hybridconnectivity/resource-manager/Microsoft.HybridConnectivity/stable/2023-03-15/hybridconnectivity.json

```diff
@@ -0,0 +1,1093 @@
+        }
+      },
+      "delete": {
+        "tags": [
+          "Endpoints"
+        ],
+        "description": "Deletes the endpoint access to the target resource.",
+        "operationId": "Endpoints_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "HybridConnectivityEndpointsDeleteDefault": {
+            "$ref": "./examples/EndpointsDeleteDefault.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceUriParameter"
+          },
+          {
+            "$ref": "#/parameters/EndpointNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The endpoint is deleted successfully."
+          },
+          "204": {
+            "description": "The endpoint has been deleted already."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/{resourceUri}/providers/Microsoft.HybridConnectivity/endpoints/{endpointName}/serviceconfigurations/{serviceConfigurationName}": {
+      "get": {
+        "tags": [
+          "Serviceconfigurations"
+        ],
+        "description": "Gets the details about the service to the resource.",
+        "operationId": "Serviceconfigurations_Get",
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "HybridConnectivityEndpointsServiceconfigurationsGetSSH": {
+            "$ref": "./examples/ServiceconfigurationsGetSSH.json"
+          },
+          "HybridConnectivityEndpointsServiceconfigurationsGetWAC": {
+            "$ref": "./examples/ServiceconfigurationsGetWAC.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceUriParameter"
+          },
+          {
+            "$ref": "#/parameters/EndpointNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ServiceConfigurationNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Service details",
+            "schema": {
+              "$ref": "#/definitions/ServiceConfigurationResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
```

Comment:
>v2 [](http://example.com/codeflow?start=71&length=2)

Optional suggestion, consider updating to reverence latest v5 ErrorResponse in case its slightly better. Probably NOT worth a breaking change if you can identify any.